src/index.ts

```diff
@@ -1 +1,4 @@
 export * from './components';
+
+//i18n setup
+export {configure} from './utils/configure';
```

Comment:
Do we need to export this helper now? What caused this new export?

Reply:
yep, we need it to be able to setup i18n locale in target project